src/app.js

```diff
@@ -1,3 +1,41 @@
 'use strict';
 
-// Write your code here
+const readline = require('node:readline');
+const { checkIsValidUserInput } = require('./modules/checkIsValidUserInput.js');
+const { generateRandomNumber } = require('./modules/generateRandomNumber.js');
+const { getBullsAndCows } = require('./modules/getBullsAndCows.js');
+
+const rl = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout,
+});
+
+const print = (message) => rl.write(message + '\n');
+
+const secretNumber = generateRandomNumber();
+
+// Функція, яка запускає одну спробу гри
+function playRound() {
+  rl.question('Введіть ваше число: ', (guess) => {
+    const validInput = checkIsValidUserInput(guess);
+
+    if (!checkIsValidUserInput(guess)) {
```

Comment:
The variable `validInput` is assigned the result of `checkIsValidUserInput(guess)`, but the same function is called again in the if condition on line 22. You can use `validInput` directly in the condition to avoid redundant function calls.